app/models/ext_management_system.rb

```diff
@@ -628,6 +646,10 @@ def build_authentication_by_role(options)
     update_authentication(creds,options)
   end
 
+  def authtype_by_role(role)
+    role == :default ? :bearer : role
```

Comment:
it's a new thing, in [L320](https://github.com/ManageIQ/manageiq/blob/master/app/models/ext_management_system.rb#L320) they do not convert from `role` to `auth type` and asume it's always the same. It creates a struct with missing `authentication` for the `default` role :-(
When using the authentication for reading data, we go to `authentications` table directly and bypass this collection methods, so we do not need this conversion there.